run/system-package/README.md

```diff
@@ -0,0 +1,13 @@
+# Cloud Run System Package sample
+
+This sample shows how to use a CLI tool installed as a system package as part of a web service.
+
+Use it with the [Using system packages tutorial](https://cloud.google.com/run/docs/tutorials/system-packages).
+
+For more details on how to work with this sample read the [Google Cloud Run Node.js Samples README](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/run).
```

Comment:
> Google Cloud Run Node.js Samples README

This phrasing is awkward. Since we're already in the Node Sample repo which is under the GCP org, can we just say something simpler like "Cloud Run Samples README"?

Reply:
There is going to be a second link added soon to the main Cloud Run Samples README in another repository.

Reply:
OK.